src/app/profiles/App/Pages/Profiles/Profile/ProfileIntro.tsx

```diff
@@ -43,8 +43,8 @@ const ProfileIntroContent = styled.section`
 
         ${Avatar} {
           margin-top: 0;
-          width: 70px;
```

Comment:
Do you know why this was malforming images ?

Reply:
Not in details but border radius 50% and size restriction of the container were responsible.